src/NServiceBus.Metrics.ServiceControl/ServiceControlMonitoring.cs

```diff
@@ -68,6 +150,46 @@ public ServiceControlMonitoringRegistration()
             }
         }
 
+        class ServiceControlReporting : FeatureStartupTask
+        {
+            public ServiceControlReporting(MetricsContext metricsContext, ISendMessages dispatcher, ReportingOptions options)
+            {
+                this.metricsContext = metricsContext;
+                this.dispatcher = dispatcher;
+                this.options = options;
+            }
+
+            protected override void OnStart()
+            {
+                HeaderValues.Add(Headers.EnclosedMessageTypes, "NServiceBus.Metrics.MetricReport");
+                HeaderValues.Add(Headers.ContentType, ContentTypes.Json);
+
+                var serviceControlReport = new NServiceBusMetricReport(dispatcher, options, HeaderValues, metricsContext);
+
+                task = Task.Run(async () =>
+                {
+                    while (cancellationTokenSource.IsCancellationRequested == false)
+                    {
+                        serviceControlReport.RunReport();
+                        await Task.Delay(TimeSpan.FromSeconds(5)).ConfigureAwait(false);
```

Comment:
Do we want to make this configurable as in v2 and v3?

Reply:
Is this the public API change that you asked about in a general PR comment?